### ISSUE TITLE
Add a "Tint View Color" setting

### ISSUE THREAD

**Issue by Geenz** [Yellow-Dog-Man/Resonite-Issues#1882]
### Is your feature request related to a problem? Please describe.

Some individuals are sensitive to bright lights.  This would help make that less problematic.

### Describe the solution you'd like

An accessibility setting that allows a user to "tint" their view with a specified color.  This should be toggleable as not all scenarios require such a setting.

### Describe alternatives you've considered

Using a facet that will do this for me.  This ~works~ but takes up facet space that I may otherwise use for other items.

**Comment by amplified1** [comment]
Could something like this also include options for more general post processing effects? Like filters to make colors more distinguishable to people with various kinds of colorblindness, or high contrast. Even effects that could just be used for fun. This might be worth a separate issue in itself, but the current method of vision filters where you just place a filter material over the user's vision sometimes has rendering issues with some materials such as unlits. If there was a generic view filter system that could also potentially allow for the user to weaken or completely disable effects as well, without it having to be implemented by creators.
